cdap-docs/integrations/source/index.rst

```diff
@@ -29,8 +29,14 @@ Integrations
 - |cloudera-ingesting|_ Ingesting and exploring data using **Impala**
 
 
+.. |cloudera-navigator| replace:: **Navigator Integration:**
+.. _cloudera-navigator: partners/cloudera/navigator-integration.html
+
+- |cloudera-navigator|_ Metadata search and discovery of CDAP entities from **Cloudera Navigator**
```

Comment:
oh no, my bad for saying Navigator. My point was that 'search' and 'discovery' are the same thing, and we're not searching metadata.